src/site/content/en/blog/fugu-status/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: New Capabilities Status
```

Comment:
```suggestion
title: New capabilities status
```

Reply:
Sentence case in titles.

(I've never read an explanation for why Google favors this. Historically, different western languages would have different rules for title capitalization. The English rules are somewhat technical, if you remember from school. It occurs to me that sentence case is easily translatable to most if not all western languages.)